### PR TITLE
Front end framing for homepage (no backend connectivity)

### PR DESCRIPTION
**Pull request by jmac522**
# Work Done

- Completed styling the homepage for anonymous and logged in users.  (Not pulling data from the backend yet). 

# Screenshots 

## Desktop (Anonymous)
logged_out_desktop

## Mobile (Anonymous)
logged_out_mobile

## Desktop (Logged In)
logged_in_desktop

## Mobile (Logged In)
logged_in_mobile

# Thoughts about linking up the back end 
- When a user is logged in, we want to show them recent reviews from any of their follows.  Currently getting this info from the back end would require getting the users follows and doing a call to the back end for each of those followers reviews and combining them.  This probably isn't a big deal in our case as we won't have huge numbers of reviews in our db.   Probably not worth making a specific endpoint for it, but just a thought. 
- We'll probably want to return albums sorted by rating order if they have been rated.  This will likely require some work on the backend. 
- Might be worth considering adding an optional `limit` param to some of our API requests to limit the # of responses for a given call, or alternately (probably easier for our case) just always limit to something like 5 on the backend by default.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,8 +33,15 @@
     "extends": [
       "react-app",
       "react-app/jest"
+    ],
+    "ignorePatterns": [
+      "node_modules/"
```
Comment (jmac522) [inline package.json:38]: Changes to package.json were to silence eslint typescript errors (it was trying to lint packages in node_modules and throwing errors).  This just ensures eslint doesn't try to lint modules that we've loaded in externally.

File: src/components/Home/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                        title="Rave"
```
Comment (jmac522) [inline src/components/Home/HomePage.js:25]: This is obviously temporary.  Figured we could finalize image assets/styling stuff once everything is linked up, so just used this random image from google image search as a placeholder.

File: package.json (hunk 1 above)
Comment (hunterrudnet) [reply package.json:38]: Got it, thanks

File: src/components/Home/HomePage.js (hunk 2 above)
Comment (hunterrudnet) [reply src/components/Home/HomePage.js:25]: Yep looks fine to me at the moment, we will deal later.

**Review by hunterrudnet — APPROVED** [review]
LGTM

**Pull request merged**